scripts/typos_rollout_check.py

```
@@ -0,0 +1,131 @@
+# requires-python = ">=3.13"
+# dependencies = []
+# ///
+"""Enforce exact phrase corrections alongside the Typos scanner."""
+
+from __future__ import annotations
+import argparse
+from dataclasses import dataclass
+from pathlib import Path
+import re
+import subprocess
+from typing import Sequence
```

Ruff (0.15.21)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)